combinations/hash.tsv

```diff
@@ -456,10 +456,10 @@ bedtools=2.31.0,samtools=1.17,ucsc-wigtobigwig=447
 r-base=4.0.2,vcftools=0.1.14-0
 perl==5.26.2,perl-sort-key=1.33,perl-yaml-libyaml=0.88
 tophat=2.1.1,samtools=1.17
-perl==5.26.2,r==3.6.0,bcftools==1.13,bedtools==2.30.0,bc==1.07.1
-python==3.7.9,pandas==1.3.4,numpy==1.21.4,pyranges==0.0.115,fire==0.4.0
-python==3.6.15,matplotlib=3.3.4,numpy=1.19.5,pyvcf=0.6.8,pysam==0.15.4,htslib==1.9
+perl=5.26.2,r=3.6.0,bcftools=1.13,bedtools=2.30.0,bc=1.07.1,grep=3	quay.io/condaforge/mambaforge:4.9.2-7	0
```

Comment:
this should be a run-time container, not a build-container. please remove the quay.io container.

Reply:
Without base image mentioned, [it ran into the error ](https://github.com/ManavalanG/multi-package-containers/actions/runs/6148876318/job/16683561548)

```
SOUT └─ grep 2**  is uninstallable because it conflicts with any installable versions previously reported.
```

Adding the base image resolved the issue.

Reply:
Please choose a different grep version that does not yield to that error. Is that possible?

Reply:
I attempted installing GNU `grep` [v3.4](https://github.com/ManavalanG/multi-package-containers/commit/5115940150e6ac51120892e70aaa9c635ff382d0) and [v2](https://github.com/ManavalanG/multi-package-containers/commit/31416d19ec2178476db165387ac97845f08e8d63), but they both failed building container (with error message shown in the previous message). It [worked when I specified a different base image](https://github.com/ManavalanG/multi-package-containers/commit/9923807646f1bd4da56b2926589f1377bea08ead).

My understanding is that runtime container includes Busybox `grep`, which is not full-featured, and it results in the error shown below. Instead I need full-featured GNU `grep`, which has the option `-P`. For more info on these two flavors of `grep`, [please see here](https://www.reddit.com/r/linuxquestions/comments/g7qs8o/what_is_the_difference_between_busyboxlinux_and).

```sh
grep: invalid option -- 'P'
BusyBox v1.32.1 (2021-04-13 11:15:36 UTC) multi-call binary.

Usage: grep [-HhnlLoqvsrRiwFE] [-m N] [-A/B/C N] PATTERN/-e PATTERN.../-f FILE [FILE]...

Search for PATTERN in FILEs (or stdin)

	-H	Add 'filename:' prefix
	-h	Do not add 'filename:' prefix
	-n	Add 'line_no:' prefix
	-l	Show only names of files that match
	-L	Show only names of files that don't match
	-c	Show only count of matching lines
	-o	Show only the matching part of line
	-q	Quiet. Return 0 if PATTERN is found, 1 otherwise
	-v	Select non-matching lines
	-s	Suppress open and read errors
	-r	Recurse
	-R	Recurse and dereference symlinks
	-i	Ignore case
	-w	Match whole words only
	-x	Match whole lines only
	-F	PATTERN is a literal (not regexp)
	-E	PATTERN is an extended regexp
	-m N	Match up to N times per file
	-A N	Print N lines of trailing context
	-B N	Print N lines of leading context
	-C N	Same as '-A N -B N'
	-e PTRN	Pattern to match
	-f FILE	Read pattern from file
```